src/xml_comment.cc

```diff
@@ -109,7 +109,7 @@ void
 XmlComment::Initialize(v8::Handle<v8::Object> target)
 {
     NanScope();
-    v8::Local<v8::FunctionTemplate> t = NanNew<v8::FunctionTemplate>(New);
+    v8::Local<v8::FunctionTemplate> t = NanNew<v8::FunctionTemplate>(static_cast<NAN_METHOD((*))>(New));
```

Comment:
Should be fixed in next patch release.
rvagg/nan@5ef813bf06c93725d03d17a91ee48d2c66dd7b3b